docs/features.rst

```diff
@@ -319,6 +319,25 @@ Automatically Flag Duplicate Findings
     and have the same CWE or title, Dojo marks the less recent finding as a duplicate. When deduplication is enabled, a
     list of deduplicated findings is added to the engagement view.
 
+Similar Findings Visualization:
+
+.. image:: _static/similar_finding_1.png
+    :alt: Similar findings list
+
+.. image:: _static/similar_finding_2.png
+    :alt: Similar findings list with a duplicate
+
+Similar Findings
+    While viewing a finding, similar findings within the same product are
+    listed along with buttons to mark one finding a duplicate of the other.
+    Clicking the 'Use as original' button on a similar finding will mark that
```

Comment:
This file is inconsistent but generally it's probably better to use a double quote (") here and below rather than a single-quote (').